docs/configuration/task-configuration.md

```diff
@@ -88,3 +88,36 @@ lambda_timeouts = {
   discover_granules_task_timeout = 300
 }
 ```
+
+## lambda_memory_sizes
+
+A configurable map of memory sizes (in MBs) for cumulus ingest module task lambdas in the form:
+
+```hcl
+<lambda_identifier>_memory_size: <memory_size>
+  type = map(string)
+```
+
+Currently the following values are supported:
```

Comment:
~Was files_to_granules left out?   It's not *really* a production lambda in that I wouldn't expect most users are utilizing it as it's really used to make test/example workflows work, but it *is* a deployed resource that comes with Core. 

There's a sensible argument for us moving it to the example project and template repo, but that's probably a separate convo. 

Same thing for fake-processing-task~


*edit nevermind, I see we also didn't add those to the timeout block, and probably had that same discussion*      I'm going to leave this alone in favor of an arch discussion.

Reply:
What is missing, however is `add_misssing_checksums` and `update-cmr-access-constraints`.   That's not an issue directly with this PR or this work, but we should probably add them to this and the timeout list.